staff/sys/firewall

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
+set -euo pipefail
+
+TMPDIR="$(mktemp -d)"
+trap 'cd / && rm -rf "$TMPDIR"' EXIT
+cd "$TMPDIR"
```

Comment:
I personally believe that apps should clean up their temporary files whenever possible. I believe that the autocleaning job should only be a backup option for those files which could not otherwise be cleaned.